elements/rh-code-block/rh-code-block.ts

```diff
@@ -9,12 +9,23 @@ import { SlotController } from '@patternfly/pfe-core/controllers/slot-controller
 import { type ColorTheme, colorContextConsumer } from '../../lib/context/color/consumer.js';
 
 import style from './rh-code-block.css';
+import type { DirectiveResult } from 'lit-html/directive.js';
 
 /* TODO
  * - style slotted and shadow fake-fabs
  * - manage state of copy and wrap, including if they are slotted. see actions.html
  */
 
+/**
+ * Returns a string with common indent stripped from each line. Useful for templating HTML
+ * @param str indented string
+ */
+function dedent(str: string) {
+  const stripped = str.replace(/^\n/, '');
+  const match = stripped.match(/^\s+/);
+  return match ? stripped.replace(new RegExp(`^${match[0]}`, 'gm'), '') : str;
+}
+
 interface CodeLineHeightsInfo {
   lines: string[];
   lineHeights: (number | undefined)[];
```

Comment:
It looks like lines 51-80 need some reformatting to appease the [Code accordion in the docs](https://deploy-preview-1837--red-hat-design-system.netlify.app/elements/code-block/code/#rh-code-block).

If you click on "Attributes" in the accordion, you can see `actions` listed as an attribute. 

![code-block-attributes-actions-docs](https://github.com/user-attachments/assets/9ef9547d-bb3e-414f-8193-d52ae41c8ac3)

Reply:
recco to either defer this to another issue or, if Dr. Mr. @coreyvickery  selects icons, make it moot